tests/translator/input/function_with_alias_and_all_properties_property.yaml

```diff
@@ -0,0 +1,20 @@
+Resources:
+  MinimalFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      AutoPublishAlias: live
+      AutoPublishAliasAllProperties: true
```

Comment:
I'd add another minimal function with this as `true` and any other property not supported by `AutoPublishAlias`, to confirm the hash changes.